src/TicketService.java

```diff
@@ -33,5 +41,11 @@ public static void main(String[] args) {
             Ticket ticket = tickets.get(i);
             System.out.println(ticket.toString());
         }
+
+        // Getting tickets by Stadium Sector 'C'
+        List<Ticket> sectorCTickets = ticketService.getTicketsByStadiumSector('C');
+        for (Ticket ticket : sectorCTickets) {
+            System.out.println(ticket.toString());
```

Comment:
As for now it is acceptable but Using System.out.println  for logging is generally not recommended

Reply:
Thank you so much! Well noted!